tests/pr_test.go

```diff
@@ -81,49 +50,27 @@ func TestRunFSCloudExample(t *testing.T) {
 			"region":                     region,
 			"existing_kms_instance_guid": permanentResources["hpcs_south"],
 			"kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+			"mongodb_version":            "4.4",
 		},
 	})
 	output, err := options.RunTestConsistency()
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
 
-func testRunComplete(t *testing.T, version string) {
+func TestRunCompleteUpgradeExample(t *testing.T) {
 	t.Parallel()
 
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
 		Testing:            t,
 		TerraformDir:       completeExampleTerraformDir,
-		Prefix:             "mongodb-comp",
+		Prefix:             "mongodb-upg",
 		BestRegionYAMLPath: regionSelectionPath,
 		TerraformVars: map[string]interface{}{
-			"mongodb_version": version,
+			"mongodb_version": "4.2",
```

Comment:
perhaps leave a code comment saying this should always lock into the lowest supported version